appbuilder/services/livesync/android-livesync-service.ts

```diff
@@ -0,0 +1,32 @@
+///<reference path="../../../.d.ts"/>
+"use strict";
+
+import { AndroidLiveSyncService } from "../../../mobile/android/android-livesync-service";
+import Future = require("fibers/future");
+
+export class AppBuilderAndroidLiveSyncService extends AndroidLiveSyncService implements IPlatformLiveSyncService {
+	constructor(private _device: Mobile.IDevice,
```

Comment:
You could declare the constructor as `private _device: Mobile.IAndroidDevice` in order to avoid casting below. I believe that wouldn't cause any problems as `IAndroidDevice` is a child of `IDevice`